evmdis/main.go

```diff
@@ -5,29 +5,96 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-
-	"github.com/Arachnid/evmdis"
+	"log"
+	"strconv"
+	"flag"
+	"github.com/soad003/evmdis"
```

Comment:
Another option is to check it out in your go build environment under my repo, then add a git remote for your fork - that way it's still compiled under the old namespace.